sql/core/src/main/scala/org/apache/spark/sql/execution/Generate.scala

```diff
@@ -34,24 +35,12 @@ import org.apache.spark.sql.catalyst.expressions._
 @DeveloperApi
 case class Generate(
     generator: Generator,
+    output: Seq[Attribute],
```

Comment:
Its too late to merge this into 1.3 since its not a regression.  So I don't see a lot of point in waiting for the correct solution.